nixos/modules/misc/version.nix

```diff
@@ -28,6 +28,8 @@ let
     DOCUMENTATION_URL = "https://nixos.org/learn.html";
     SUPPORT_URL = "https://nixos.org/community.html";
     BUG_REPORT_URL = "https://github.com/NixOS/nixpkgs/issues";
+  } // lib.optionalAttrs (cfg.variant_id == null) {
```

Comment:
Should we not invert the condition here? We want to include it if `cfg.variant_id != null`, right?

Reply:
:)